File: 20190605/387. First Unique Character in a String.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        mystr = 'abcdefghijklmnopqrstuvwxyz';
        l = [s.index(c) for c in mystr if s.count(c) == 1]
        return min(l) if len(l) > 0 else -1

File: 20190605/387. First Unique Character in a String.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        unordered_map<char, int> mymap;
        for(auto c : s) mymap[c] += 1;
        for(int i = 0; i < s.size(); i++)
        {
            if (mymap[s[i]] == 1)
            {
                return i;
            }
        }
        return -1;
        
    }
};